goaviatrix/const.go

```diff
@@ -14,9 +14,11 @@ const (
 	AWSChina    = 1024
 	AzureChina  = 2048
 	AliCloud    = 8192
-	EDGECSP     = 65536
-	EDGEEQUINIX = 524288
-	EDGENEO     = 262144
+	AWSTS       = 16384  // AWS Top Secret Region (C2S)
+	AWSS        = 32768  // AWS Secret Region (SC2S)
```

Comment:
I believe these were just removed recently in another PR since they are not longer supported, we should remove this  from the PR.